backend/catalogue/models.py

```diff
@@ -114,7 +114,50 @@ def server_info(self):
         r = requests.get(url=url, params=params, verify=False)
 
         return r.json()
-    
+
+    def cql_property_values(self, cql_properties: list) -> dict:
+        params = {
+            'service': 'WFS',
+            'version': '2.0.0',
+            'request': 'GetFeature',
+            'typeNames': self.layer_name,
+            'outputFormat': 'application/json',
+            'propertyName': f"({','.join(cql_properties)})",
+        }
+        r = requests.get(url=self.server_url, params=params, verify=False)
```

Comment:
I'm trying to decide how I feel about making http requests from a model method.  It's a bit "unexpected" (db queries is obviously standard; http might not be).  *this is  not thought through:* you could perhaps separate out the query and the processing; maybe a separate method to create the query and this one to parse it, or perhaps have this method take the "http queryer" that's then used by this method (basically, passing in requests, but it could be a bit abstracted.  It makes it clear to the caller that it'll be doing a network request).  Both of those options would make it simpler to test, as well, which is often a useful sniff-test.